roles/multus-crd/templates/crd.yml.j2

```diff
@@ -0,0 +1,22 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
General comment on some of these templates. If we aren't templating anything, would it be better to just make these files that we `copy` onto the machine? I guess this is a bit of future proofing if you wanted to actually add variables in here at some point though. Not a big deal.

Reply:
Good call, but, I kinda figured these might get extended. Maybe it's YAGNI, but, if the scenarios expand, these will definitely change. The first iteration here will be like two configs that are static just to show "cool it works." but anything further these would get templated. appreciate the thought into it.